tools/migration/main.go

```diff
@@ -97,6 +108,22 @@ func getVerbose() bool {
 	return res
 }
 
+func getLogFile() (*os.File, error) {
+	path, err := getLogFilePath()
+	if err != nil {
+		return nil, err
+	}
+	return os.OpenFile(path, os.O_APPEND|os.O_CREATE, 0644)
+}
+
+func getLogFilePath() (string, error) {
+	if logPath, found := findOpt("--log-file", os.Args); found {
+		return homedir.Expand(logPath)
```

Comment:
We shouldn't need to expand values from args – the shell will do it. The only reason we should be expanding ourselves is when expanding a hardcoded path.

Reply:
Ah thanks, overreacting after the last bug 😅